packages/distribution/addon/components/cd-inquiry-dialog.js

```diff
@@ -68,11 +70,42 @@ export default class CdInquiryDialogComponent extends Component {
       if (allWorkItems.edges.every((edge) => edge.node.status === "CANCELED")) {
         this.router.transitionTo("index");
       }
+
+      /**
+       * Get work items that were redoable in the previous calulation and are
+       * now not anymore. This indicates that the work item was redone which
+       * should result in a transition to the answer view.
+       */
+      const redoableChangedToTrue = this._redoableStates.find((stateObj) => {
+        const updatedWorkItem = allWorkItems.edges.find(
+          (edge) => decodeId(edge.node.id) === stateObj.id
+        );
+
+        return (
+          stateObj.isRedoable && !(updatedWorkItem?.node.isRedoable ?? true)
```

Comment:
Which case is handled by the `?? true`? Is the `isRedoable` property reset in certain work-items?

Reply:
If `updatedWorkItem` is `undefined`

Reply:
Ah yes, makes sense, thanks.

Reply:
Minor nitpick: Shouldn't the name be `redoableChangedToFalse` or `redoneWorkItem`, if we're looking for work-items that are no longer redoable and have been redone?

Reply:
Valid point, changed it to `redoneWorkItem`